build.sbt

```diff
@@ -213,8 +216,8 @@ lazy val swagger = project
     circe
   )
 
-lazy val akkaHttp = project
-  .in(file("modules/akkaHttp"))
+lazy val `akka-http` = project
```

Comment:
I don't see this as an improvement.
My practice shows that backticked names are cumbersome in development somehow.

Reply:
@Odomontois Quick question: Do u prefer symbol literals to strings?